src/screens/Dashboard/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect, useState } from 'react';
+
+import { getTokens } from 'helpers/authentication';
+
+const DashBoardScreen = (): JSX.Element => {
```

Comment:
Are we missing unit test for this screen?

Reply:
@hoangmirs this screen is not completed so I decided to write unit tests in the Integrate test. Please skip it 🙆‍♂️